### ISSUE TITLE
`currentSchema` support

### ISSUE THREAD

**Issue by vitaly-t** [iceddev/pg-connection-string#9]
Introduced in PostgreSQL 9.4, it allows the following syntax:

```
postgres://username:password@host:port/database?currentSchema=myschema
```

which needs to be added.

See also: http://stackoverflow.com/questions/4168689/is-it-possible-to-specify-schema-when-connecting-to-postgres-with-jdbc

**Comment by elmigranto** [comment]
Looks like querystring parsing only supports certain limited number of options. There are a lot more of them: https://www.postgresql.org/docs/9.5/static/libpq-connect.html#LIBPQ-CONNSTRING.

Perhaps we should pass in everything we parsed, and not a subset.

(Relevant: https://github.com/brianc/node-postgres/issues/1095)

**Comment by elmigranto** [comment]
Should be solved with #7.

**Comment by baclv** [comment]
```ts
import { parse, ConnectionOptions } from 'pg-connection-string';

interface customConnectionOptions extends ConnectionOptions {
  schema: string;
}

const db = parse(process.env.DATABASE_URL) as customConnectionOptions;

console.log(db)
```
